programs/svm-spoke/src/instructions/slow_fill.rs

```diff
@@ -206,13 +206,13 @@ pub struct ExecuteV3SlowRelayLeaf<'info> {
     pub system_program: Program<'info, System>,
 }
 
-pub fn execute_v3_slow_relay_leaf<'info>(
-    ctx: Context<'_, '_, '_, 'info, ExecuteV3SlowRelayLeaf<'info>>,
+pub fn execute_slow_relay_leaf<'info>(
+    ctx: Context<'_, '_, '_, 'info, ExecuteSlowRelayLeaf<'info>>,
     slow_fill_leaf: Option<V3SlowFill>,
     proof: Option<Vec<[u8; 32]>>,
 ) -> Result<()> {
     let ExecuteV3SlowRelayLeafParams { slow_fill_leaf, proof, .. } =
-        unwrap_execute_v3_slow_relay_leaf_params(slow_fill_leaf, proof, &ctx.accounts.instruction_params);
+        unwrap_execute_slow_relay_leaf_params(slow_fill_leaf, proof, &ctx.accounts.instruction_params);
```

Comment:
This seems bit inconsistent that we rename the internal function to `unwrap_execute_slow_relay_leaf_params`, but keep the same `ExecuteV3SlowRelayLeafParams` struct name when deconstructing its parameters.